model/task_lifecycle.go

```diff
@@ -260,7 +260,7 @@ func activatePreviousTask(ctx context.Context, taskId, caller string, originalSt
 
 	// If this is a generator task and we originally were stepping back a generated task, activate the generated task
 	// once the generator finishes.
-	if prevTask.GenerateTask && originalStepbackTask != nil {
+	if prevTask != nil && prevTask.GenerateTask && originalStepbackTask != nil {
```

Comment:
Seems like we could just return nil early on L248 if prevTask is nil and avoid all the `prevTask != nil` checks, but it's not a big deal.